src/sanket/ArrayOperationsEx6.java

```diff
@@ -0,0 +1,71 @@
+/*Assignment - 25 : 20th Sep'2022
+
+Program 1: Print all the repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+output : Hindavi 
+         Shubham 
+
+Program 2: Print all the non repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham hindavi Shubham Titiksha Swati";
+output : Maulik 
+         Puja
+         Titiksha
+         Swati
+*/
+
+package sanket;
+
+public class ArrayOperationsEx6 {
+
+	void getRepeatString(String arr) {
+
+		String[] tempInput = arr.split(" ");
+
+		for (int index = 0; index < tempInput.length; index++) {
+			String tempWord = tempInput[index];
+			int count = 0;
+			for (int j = index + 1; j < tempInput.length; j++) {
+				if (tempWord.equals(tempInput[j])) {
+					count++;
+					break;
+				}
+			}
+			if (count > 0) {
+				System.out.println(tempInput[index]);
+			}
+		}
+	}
+	
+	void getNonRepeatString(String arr) {
```

Comment:
Here you can also use the 
for(int i = 0;i<input.length;i++){
if(str.indexOf(input[i]) == str.lastIndexOf(input[i])){
System.out.println(input[i]);
}
}

Reply:
Thanks Harsh for you alternate solution. I will impalement this if same form of question asked.